server/api/card.js

```diff
@@ -81,18 +78,24 @@ router.post('/make25/forRoom/:roomId', async (req, res, next) => {
       },
     });
 
+    // Sanity check that the POSSIBLE words we will pull from were
+    if (!allWords) return res.sendStatus(404);
```

Comment:
I started putting in these sanity checks so, if something does go wrong, atleast well get a 404 instead of a 500